paddle/fluid/framework/details/execution_strategy.h

```diff
@@ -19,12 +19,13 @@ namespace framework {
 namespace details {
 
 struct ExecutionStrategy {
-  enum ExecutorType { kDefault = 0, kExperimental = 1 };
+  enum ExecutorType { kDefault = 0, kExperimental = 1, kDoubleQueue = 2 };
```

Comment:
Document the difference of various strategy?

Reply:
Do you make speed comparison on some model?